sdk/go/protos/feast/core/Store.pb.go

```diff
@@ -296,6 +301,20 @@ func (m *Store_RedisConfig) GetPort() int32 {
 	return 0
 }
 
+func (m *Store_RedisConfig) GetInitialBackoffMs() int32 {
```

Comment:
There was a bit of codegen that was stale here I guess. Also a few Go module version updates, I think the Go SDK is kind of dormant so those aren't significant.